Mercury-Store/Mercury-Store/Profile/Models/ProfileModel.swift

```diff
@@ -6,3 +6,33 @@
 //
 
 import Foundation
+
+struct Customers {
+    let customers: [Customer]
+}
+
+// MARK: - Customer
+struct Customer  {
+    let id: Int
+    let email: String
+    let createdAt, updatedAt: Date
+    let firstName, lastName , tags: String
+    let ordersCount: Int
+    let totalSpent: String
+    let lastOrderID: Int?
+    let verifiedEmail: Bool
+    let phone: String
+    let lastOrderName: String?
+    let addresses: [Address]
+    let defaultAddress: Address
```

Comment:
Why most of the properties are not optional?!
I think this is added to be decoded from server, for cases like this we need to make sure that property is optional or not, defaultAddress for example.

Reply:
resolved